src/v/kafka/server/requests.cc

```diff
@@ -225,6 +225,19 @@ bool track_latency(api_key key) {
     }
 }
 
+// only track partition mutation quotas for create/delete topics & create
+// partitions requests
+bool track_partition_mutations(api_key key) {
+    switch (key) {
+    case create_topics_handler::api::key:
```

Comment:
I know this is mirroring the KIP-599 list, but I wonder if we should take the opportunity to define a broader definition.  For example, we could include topic config changes.

It would be comforting to know that we had a relatively broad throttle on Kafka operations that can result in controller log appends, because we currently don't compact the controller log, and rely on the overall number of entries being "reasonable"

Reply:
Interesting, yeah we don't have to strictly adhere to the KIP at all. Are topic config changes equally as expensive as partition mutations?

Reply:
Not quite equally (creating partitions is inherently more expensive in terms of going out there and creating log segments etc), but they're both things that can cause serious problems if we let users flood us.

Maybe we should apply the rate limit to anything that's a controller log write?  That would include things like users and acls.  What do you think @mmaslankaprv ?

Reply:
FWIW Kafka addresses this by tracking different quotas and having a manager for each, due to the fact that not all requests are equally as expensive for the controller to handle.